<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="4,6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QListView" name="listView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,5">
        <item>
         <widget class="PlotWidget" name="mainSoundPlot" native="true"/>
        </item>
        <item>
         <widget class="PlotWidget" name="soundPlot" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,6">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,3,8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1,10">
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>I</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>U</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <widget class="QVideoWidget" name="widget_3" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Project</string>
    </property>
    <widget class="QMenu" name="menuRecentProject">
     <property name="title">
      <string>Open Recent</string>
     </property>
    </widget>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="menuRecentProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentFile">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Open Recent</string>
     </property>
    </widget>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="menuRecentFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveAsFile"/>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSubMyth_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_SubMyth"/>
    <addaction name="actionAbout_SubAi"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPrefrences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFile_2"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewProject">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAsFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionSubMyth_Help">
   <property name="text">
    <string>SubMyth Help</string>
   </property>
  </action>
  <action name="actionAbout_SubMyth">
   <property name="text">
    <string>About SubMyth</string>
   </property>
  </action>
  <action name="actionAbout_SubAi">
   <property name="text">
    <string>About SubAi</string>
   </property>
  </action>
  <action name="actiona">
   <property name="text">
    <string>a</string>
   </property>
  </action>
  <action name="actionPrefrences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5/QtMultimediaWidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
